gatt_blacklist.txt

```diff
@@ -19,6 +19,8 @@
 00001530-1212-efde-1523-785feabcd123
 # TI's Over-the-Air Download service, http://www.ti.com/lit/ug/swru271g/swru271g.pdf:
 f000ffc0-0451-4000-b000-000000000000
+# Cypress's Bootloader service, http://www.cypress.com/file/175561/download:
```

Comment:
I think we should explain this URL more, similar to the way Francois did.

CYPRESS BOOTLOADER SERVICE_001-97547.pdf available at
http://www.cypress.com/file/175561/download with and account required, linked from
http://www.cypress.com/documentation/software-and-drivers/cypresss-custom-ble-profiles-and-services